samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccountName: bookinfo-details
       containers:
       - name: details
-        image: docker.io/istio/examples-bookinfo-details-v1:1.17.0
+        image: istiopetproject.azurecr.io/examples-bookinfo-details-v1:latest
```

Comment:
Unfortunately, we will have issues in case of using latest tag. Please think about how to use unique tag instead for deployment. It might be some id or hash